partner_invoicing_mode_fourteen_days/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2020 Camptocamp
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Partner Invoicing Mode Fourteen Days",
```

Comment:
I prefer the name fortnightly

```suggestion
    "name": "Partner Invoicing Mode Fortnightly",
```